packages/dds/map/api-report/map.api.md

```diff
@@ -44,7 +44,7 @@ export interface ICreateInfo {
     csn: number;
 }
 
-// @alpha
+// @internal
```

Comment:
Is it expected for items to become more private in this PR?  I would have expected this one to remain `@alpha` at least?